core/inc/knoting/shape.h

```diff
@@ -2,16 +2,19 @@
 #include <PxPhysicsAPI.h>
 #include <knoting/components.h>
 #include <knoting/px_variables_wrapper.h>
+#include <knoting/rigidbody.h>
 
 namespace knot {
 namespace components {
+
 class Shape {
```

Comment:
we may have to potentially rename this class in the future to 'PhysicsCollider' or 'PhysicsShape' as 'Shape' is an ambiguous class name

Reply:
I absolutely agree!